test/team_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+require './lib/player'
+require './lib/team'
+
+class TeamTest < Minitest::Test 
+  
+  def test_it_exists 
+    player = Team.new("France")
+    
+    assert_instance_of Team, player
+  end
+  
+  def test_it_has_attributes
+    skip
+    france = Team.new("France")
+    player_1 = Player.new("Paul Pogba", :midfielder)
+    
+    assert_equal "France", france.country
+    assert_equal [], france.players
+    assert_equal "Paul Pogba", france.name
```

Comment:
You don't need to test these attributes here. These are attributes of the Player, not the Team. Remember, this should be a test for the Team class.